product_harmonized_system/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+
+============================================
+Harmonized System Codes (and National Codes)
+============================================
+
+This module contains the objects for Harmonised System Codes (H.S. codes). The full nomenclature is available from the `World Customs Organisation <http://www.wcoomd.org/>`. These code are usually required on the Proforma invoices that are attached to the packages that are shipped abroad.
+
+This module also handle the local/national extensions to the H.S. codes. The import of the full nomenclature is not provided by this module ; it should be provided by localization modules.
+
+You will also be able to configure the country of origin of a product, which is often required on the proforma invoice for the customs.
+
+This module should be usefull for all companies that export physical goods abroad. This module is also used by the Intrastat modules for the European Union, cf the *intrastat_product* module.
+
+
+Installation
+============
+
+This module is NOT compatible with the *report_intrastat* module from the official addons.
+
+
+Usage
+=====
+
+To create H.S. codes, go to the menu *Sales > Configuration > Product Categories and Attributes > H.S. Codes*.
+
```

Comment:
Incorrect for Odoo 12.0.
Also this module doesn't depend on Sales hence no Sales / Configuration menu available.
Please adapt.